### DISCUSSION TITLE
Table UI will not allow update with a generated tsvector column

### DISCUSSION THREAD

**Discussion by GaryAustin1** [supabase/supabase#3290] [Questions] [answered]
Not sure if I'm doing something wrong, or an issue with the UI table manager.  I create a full text tsvector column per the guides with SQL editor.  I then try to change the title in the table UI and get "column "title_search" can only be updated to DEFAULT".
Works with an SQL update and inserting a row with the Table UI.  Guessing it is trying to update that column directly on row change but not sure.

**Comment by GaryAustin1** [comment, marked as answer]
Well I've discovered my IDE (JetBrains) had a built in Database manager plugin that works with the Postgres SupaBase cloud connection, so this is no longer an issue for me.  Coming from Firebase/Javascript and before that CMS's did not even think to look for an external tool versus the builtin SupaBase UI.  So much easier now to set up, experiment, watch results.  It even knows about geography fields and can display table rows on a map! :)

**Reply by fergusmeiklejohn** [reply]
DataGrip plugin is great..
